### PR TITLE
Add sample datasets

### PR DESCRIPTION
**Pull request by mathidachuk**
Addresses #11 

Please add org chart dataset!

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+#' Source: \url{https://github.com/allenai/hierplane/}
+"styles"
+
+#' Starwars Films and Starships
+#'
+#' A sample hierplane dataframe derived from the {dplyr} `starwars` dataset that
+#' shows the appearances of each starship in the first seven Star Wars movies.
+#'
+#' Source: \url{https://dplyr.tidyverse.org/reference/starwars.html}
+"hp_starwars"
```
Comment (tyluRp) [inline R/data.R:14]: Oooo, what if we ditch the `hp_` and just call this `starships`?

Comment (mathidachuk) [reply R/data.R:14]: SMART

Comment (tyluRp) [reply R/data.R:14]: hahahahahaha 😆

**Comment by tyluRp** [comment]
I swear I am trying! Unnesting json is hard 😢 

``` r
library(jsonlite)
library(tibble)
library(tidyr)
library(dplyr)

df <- "/Users/tylerlittlefield/Desktop/organization.json" %>% 
  read_json()

df$root %>% 
  as_tibble() %>% 
  unnest_longer(attributes) %>% 
  unnest_longer(children)
#> # A tibble: 24 x 5
#>    word          attributes nodeType children     children_id
#>    <chr>         <chr>      <chr>    <named list> <chr>      
#>  1 Esther Walker CEO        root     <chr [1]>    word       
#>  2 Esther Walker CEO        root     <chr [1]>    link       
#>  3 Esther Walker CEO        root     <list [1]>   attributes 
#>  4 Esther Walker CEO        root     <chr [1]>    nodeType   
#>  5 Esther Walker CEO        root     <chr [1]>    word       
#>  6 Esther Walker CEO        root     <chr [1]>    link       
#>  7 Esther Walker CEO        root     <chr [1]>    nodeType   
#>  8 Esther Walker CEO        root     <list [4]>   children   
#>  9 Esther Walker CEO        root     <chr [1]>    word       
#> 10 Esther Walker CEO        root     <chr [1]>    link       
#> # … with 14 more rows
```

<sup>Created on 2020-07-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by tyluRp** [comment]
Merging, we can continue to submit PR's for additional datasets when we have them available.

**Pull request merged**